<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportRegisterContractsSetupDialog</class>
 <widget class="QDialog" name="ReportRegisterContractsSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="lblFinanceType">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblContract">
     <property name="text">
      <string>Договор</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="edtEndDate"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="edtBegDate"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="CRBComboBox" name="cmbFinanceType"/>
   </item>
   <item row="6" column="1">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="1">
    <widget class="CContractComboBox" name="cmbContract"/>
   </item>
   <item row="5" column="1">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblDateSortType">
     <property name="text">
      <string>Выводить по</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="btnToMonth">
       <property name="text">
        <string>месяцам</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnToDay">
       <property name="text">
        <string>дням</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CContractComboBox</class>
   <extends>QComboBox</extends>
   <header>Accounting/ContractComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportRegisterContractsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportRegisterContractsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
